docs/api/resource.rst

```diff
@@ -1,28 +1,275 @@
+and ``If-None-Match`` headers (see :ref:`section about timestamps <server-timestamps>`).
+
+**Request**:
+
+.. code-block:: http
+
+    GET /v1/buckets/syncto/collections/history/records HTTP/1.1
+    Authorization: BrowserID eyJhbGciOiJSUzI1NiJ9...FHGg
+    Host: syncto.dev.mozaws.net
+    User-Agent: HTTPie/0.9.2
+    X-Client-State: 64e8bc35e90806f9a67c0ef8fef63...
+
+
+**Response**:
+
+.. code-block:: http
+
+    HTTP/1.1 200 OK
+    Access-Control-Expose-Headers: Content-Length, Quota-Remaining, Alert, Retry-After, Last-Modified, Total-Records, ETag, Backoff, Next-Page
+    Content-Length: 1680
+    Content-Type: application/json; charset=UTF-8
+    Date: Tue, 06 Oct 2015 13:57:24 GMT
+    ETag: "1442849064460"
+    Last-Modified: Mon, 21 Sep 2015 15:24:24 GMT
+    Total-Records: 6
+
+    {
+        "data": [
+            {
+                "id": "VLkOS7iT5C94",
+                "last_modified": 1441868927070,
+                "payload": "{\"ciphertext\":\"Wf2AoZiOly...\",\"IV\":\"jW7JFPf...\",\"hmac\":\"989352d9b5e0c6...\"}",
+                "sortindex": -1
+            },
+            {
+                "id": "qYYobAN_p9vS",
+                "last_modified": 1441868927070,
+                "payload": "{\"ciphertext\":\"3upjoLrO...7\",\"IV\":\"3O/nPq82xUT...\",\"hmac\":\"addce0f9d3024ed9fd0042b...\"}",
+                "sortindex": 100
+            },
+    		...
+        ]
+    }
+
+
+Filtering and Sorting
+---------------------
+
+Filtering properties of the Firefox Sync protocol are exposed as well
+as the one needed by Kinto clients.
+
+- ``_since`` with the ETag value to fetch changes from the previous
+  time.
+- ``_sort`` can be either ``newest`` or ``index`` or ``-last_modified`` and ``-sortindex``.
+- ``_limit`` to limit the number of items per pages (no limit by default).
+- ``ids`` to define the list of requested records.
+
+
+Pagination
+----------
+
+The ``Next-Page`` header will be sent to you with the URL to fetch the
+next page. It will include defined ``_limit`` and ``_token`` values
+automatically.
+
+When the ``Next-Page`` is not present, it means that you fetched
+everything already.
+
+
+Counting
+--------
+
+Contrary to what Kinto does, the ``Total-Records`` only contains the
+number of records in the current request
+`for now <https://github.com/mozilla-services/syncto/issues/43>`_.
+
+You may ask the request without the ``_limit`` parameter to get all
+the records at once.
+
+
+Polling for changes
+-------------------
+
+The ``_since`` parameter is provided as an alias for ``gt_last_modified``.
+
+If the request header ``If-None-Match`` is provided as described in
+the :ref:`section about timestamps <server-timestamps>` and if the
+collection was not changed, a ``304 Not Modified`` response is returned.
+
+
+Additionnal headers
+-------------------
+
+The ``Quota-Remaining`` header is not part of the Kinto protocol yet
+but is passed through if present in Firefox Sync responses.
+
+Its value is in kB.
```

Comment:
*Kilobyte* is one word (no space).

As an aside, this is a surprisingly tricky term! `kB` vs `KB`; `kilo` vs `kebi`, etc.
https://en.wikipedia.org/wiki/Kilobyte